hexa/pipelines/models.py

```diff
@@ -179,9 +206,13 @@ def is_schedulable(self):
     def is_latest_version(self):
         return self == self.pipeline.last_version
 
+    @property
+    def version_name(self):
+        return str(self.version_number) + (f" [{self.name}]" if self.name else "")
```

Comment:
Not sure about this format ? Any ideas ?

Reply:
I would assume that if the user wants to add a name, it has to be more important than the version number so I would invert the two to put the name in front and the number in second. WDYT?

Reply:
I think keeping the version number in square brackets sets the right tone that it is autogenerated descriptive number

Reply:
@nazarfil do you mean like this ?

- ABC [v1]
- ABCD [v2]
- v3
- ABCDE [v4]

I agree that having the generated number displayed with a slight style difference can bring its autogenerated nature.

Reply:
Thanks all for the suggestions, I changed it in the latest version